### PR TITLE
components and pages UI

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
- created components UI in react
- created all pages UI in react

### CODE REVIEW

File: src/components/navbar/navbar.jsx [hunk 1]
```diff
@@ -2,7 +2,44 @@ import React from 'react'
 
 const Navbar = () => {
   return (
-    <div>Navbar</div>
+    <>
+    <nav className="navigation container nav-bar">
+  <a href="../index.html" className="links links-mr">
```
Comment (srejitk) [inline src/components/navbar/navbar.jsx:7]: Make use of Links and routes instead of a tags

File: src/components/card/card.jsx [hunk 2]
```diff
@@ -1,8 +1,41 @@
...
+            </div>
+            </div>
+
+            <div className="card-price flex-row space-between">
+              <div className="new">Rs 1000</div>
+              <div className="discounted">Rs5000</div>
+            </div>
+            <div class="card-buttons flex-col gap-sm">
+            <a href="#" class="links primary-link stack">Add to cart</a>
+            <a href="#" class="links links-secondary stack">Wishlist</a>
```
Comment (nishasen) [inline src/components/card/card.jsx:34]: Add links here later, instead of a tag

**Review by nishasen — APPROVED** [review]
Try making some more components which you can reuse in various pages.

**Review by tanishq20 — APPROVED** [review]
Code looks clean and easy to understand.

**Review by srejitk — APPROVED** [review]
Convert more elements into components so that you can reuse them later on.

Also add routing ASAP.

File: src/components/card/card.jsx (hunk 2 above)
Comment (Priyanshi-Tiwari) [reply src/components/card/card.jsx:34]: will do

File: src/components/navbar/navbar.jsx (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply src/components/navbar/navbar.jsx:7]: okay...will do

**Pull request merged**